packages/data-stores/src/domain-suggestions/index.ts

```diff
@@ -10,29 +10,29 @@ import { STORE_KEY } from './constants';
 import reducer, { State } from './reducer';
 import * as actions from './actions';
 import * as resolvers from './resolvers';
-import * as selectors from './selectors';
+import createSelectors, { Selectors } from './selectors';
 import { DispatchFromMap, SelectFromMap } from '../mapped-types';
 import { controls } from '../wpcom-request-controls';
 
 export * from './types';
 export { State };
 
 let isRegistered = false;
-export function register(): typeof STORE_KEY {
+export function register( vendor: string ): typeof STORE_KEY {
```

Comment:
One idea: if we follow a similar pattern for other stores (accept some arg upon initialization), how about turning those args into named ones? So it's better documented (and doesn't just depend on arg order) what they're for.

```suggestion
export function register( { vendor } : { vendor: string } ): typeof STORE_KEY {
```